src/components/footer/index.tsx

```diff
@@ -1,10 +1,8 @@
 import { Container } from '../container'
-import { Footer } from './styles'
+import { StyledFooter } from './styles'
 
-export const FooterComponent = () => {
-  return (
-    <Footer>
-      <Container></Container>
-    </Footer>
-  )
-}
+export const Footer = () => (
+  <StyledFooter>
+    <Container></Container>
+  </StyledFooter>
+)
```

Comment:
What is the reason of creation this footer?

Reply:
Corrected